core/templates/dev/head/pages/exploration-editor-page/exploration-editor-page.controller.ts

```diff
@@ -17,6 +17,7 @@
  *               help tab in the navbar.
  */
 
+require('d3');
```

Comment:
Can we maybe move it to the place where it is actually needed? (if it is possible to find that place)

Reply:
I found a way to do this and I've done it.